### PR TITLE
Add new functionality for profile

### PR DESCRIPTION
**Pull request by chenxidev1129**

### CODE REVIEW

**Comment by kgg0305** [comment]
@chenxidev1129 How did you manage to authenticate the user?

**Comment by chenxidev1129** [comment]
> @chenxidev1129 How did you manage to authenticate the user?

`var userId = User.FindFirst(ClaimTypes.NameIdentifier)?.Value;`

I implemented this code for that.

**Comment by chenxidev1129** [comment]
@johnsmith0305 @fullstackdev0305 Who can review this PR?

**Review by johnsmith0305 — CHANGES_REQUESTED** [review]
We need to create new endpoint to upload profile image.

**Comment by johnsmith0305** [comment]
We need to create a new endpoint to show image with image file name.

**Comment by chenxidev1129** [comment]
@johnsmith0305 I fixed all

File: taskmaster-api/Services/Interface/IProfileService.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+using Microsoft.AspNetCore.Mvc;
```
Comment (johnsmith0305) [inline taskmaster-api/Services/Interface/IProfileService.cs:1]: I am not sure why this is used

**Review by johnsmith0305 — CHANGES_REQUESTED** [review]

**Comment by smartwebdev2023** [comment]
I formatted/refactored some code.

**Review by smartwebdev2023 — APPROVED** [review]

**Review by johnsmith0305 — APPROVED** [review]

**Review by fullstackdev0305 — APPROVED** [review]

**Pull request merged**